adapter/internal/operator/apis/dp/v1alpha1/apipolicy_types.go

```diff
@@ -60,10 +60,10 @@ type CORSPolicy struct {
 	AccessControlMaxAge           *int     `json:"accessControlMaxAge,omitempty"`
 }
 
-// InterceptorReference holds InterceptorService reference using name and namespace
+// InterceptorReference holds InterceptorService references for the InterceptorService resource.
 type InterceptorReference struct {
-	// Name is the referced CR's name of InterceptorService resource.
-	Name string `json:"name"`
+	// Refs is an array of referenced Interceptors.
+	Refs []string `json:"refs,omitempty"`
```

Comment:
Let's not change it to Refs. let's keep the crd as it is. because it was designed after considering future changes as well.